### DISCUSSION TITLE
Youtube music no longer working on latest MacOs

### DISCUSSION THREAD

**Discussion by smakus** [jagrosh/MusicBot#1539] [Troubleshooting]
Youtube playback is no longer working on the latest JMusicBot (4.0.4) and macOS (14.4.1).

I'm not sure if these warnings are relevant, but they could be:

(terminal output when initially running the jar):

 WARNING: Secure coding is automatically enabled for restorable state! However, not on all supported macOS versions of this application. Opt-in to secure coding explicitly by implementing NSApplicationDelegate.applicationSupportsSecureRestorableState:.

Console output from the bot:

[10:38:26] [INFO] [YoutubeAccessTokenTracker]: Updating YouTube visitor id (current is CgsxTUN3Y19XeGVJMCj2rMuwBjIKCgJVUxIEGgAgNToKIPmp2oblzrWJZg%3D%3D).
[10:38:26] [INFO] [YoutubeAccessTokenTracker]: Updating YouTube visitor id succeeded, new one is Cgt0NWNJWklFN0RoUSiStMuwBjIKCgJVUxIEGgAgGzoKINW4jqunwraJZg%3D%3D, next update will be after 600 seconds.

I believe the console output is normal, and the token will refresh after 10 minutes, however no music ever plays and the song is immediately done without ever playing.

Soundcloud playback still works (forcing it to play through Soundcloud by using the scsearch command).

**Comment by BlasterM5** [comment]
Happens in Ubuntu. I don't believe it's OS specific.

**Comment by jagrosh** [comment]
Duplicate of #1537
